File: src/main/java/com/gluck/jobtracker/JobtrackerApplication.java
package com.gluck.jobtracker;

import com.vaadin.flow.component.dependency.StyleSheet;
import com.vaadin.flow.theme.lumo.Lumo;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@StyleSheet(Lumo.STYLESHEET)
@StyleSheet(Lumo.UTILITY_STYLESHEET)
@StyleSheet("styles.css")
public class JobtrackerApplication {
    public static void main(String[] args) {
        SpringApplication.run(JobtrackerApplication.class, args);
    }
}

File: src/main/kotlin/com/gluck/jobtracker/JobtrackerApplication.kt
package com.gluck.jobtracker

import com.vaadin.flow.component.dependency.StyleSheet
import com.vaadin.flow.theme.lumo.Lumo
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
@StyleSheet(Lumo.STYLESHEET)
@StyleSheet(Lumo.UTILITY_STYLESHEET)
@StyleSheet("styles.css")
class JobtrackerApplication

    fun main(args: Array<String>) {
        runApplication<JobtrackerApplication>(*args)
    }

